pyproject.toml

```
@@ -0,0 +1,118 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "django-lasuite"
+version = "0.0.1"
+description = "Django La Suite - A Django library"
+readme = "README.md"
+requires-python = ">=3.10"
+license = {file = "LICENSE"}
+authors = [
+    {name = "DINUM", email = "dev@mail.numerique.gouv.fr"},
+]
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "Framework :: Django",
+    "Framework :: Django :: 4.2",
```

```suggestion
    "Framework :: Django :: 5.0",
```

This is the version used in the dependencies section.